### PR TITLE
php-ref 0.5.0

### PR DESCRIPTION
**Pull request by pinepain**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

-----

`php70-ref` should be dropped in a while as I removed PHP 7.0 support.

### CODE REVIEW

File: Formula/php70-ref.rb [hunk 1]
```diff
@@ -23,4 +23,11 @@ def install
     prefix.install "modules/ref.so"
     write_config_file if build.with? "config-file"
   end
+
+  def caveats
+    <<-EOT.undent
```
Comment (SMillerDev) [inline Formula/php70-ref.rb]: Isn't this supposed to be indent?

Comment (pinepain) [reply Formula/php70-ref.rb]: I guess it doesn't matter.

Comment (SMillerDev) [reply Formula/php70-ref.rb]: magic..

**Comment by SMillerDev** [comment]
@BrewTestBot test this please

**Comment by SMillerDev** [comment]
Thanks @pinepain

**Pull request closed without merging**